shoplist/src/main/java/com/netguru/codereview/network/model/ShopListResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+data class ShopListResponse(
+    val list_id: String,
```

Comment:
It is good to always use the same variable naming convention (in this case camelCase which is the suggested one for kotlin vals)
```suggestion
    val listId: String,
```